lib/python/ensembl/brc4/runnable/process_gff3.py

```diff
@@ -78,6 +78,13 @@ def param_defaults(self):
                 "long_terminal_repeat",
                 "STS"
             ),
+            "ignored_transcript_types": (
+                    "antisense_RNA",
+                    "RNase_MRP_RNA",
+		            "3'UTR",
+                    "5'UTR",
+		            "misc_RNA"
```

Comment:
Don't we load misc_RNAs?

Reply:
yes, we do, according to the code here is "misc_RNA" only accepted as gene_types?